Day7-Assignment/src/style.css

```diff
@@ -0,0 +1,66 @@
+body{
+    margin:0px;
+}
+#class-container{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+a{
+    text-decoration: none;
+    color:black;
+}
+
+#card{
+    width: 25%;
+    border: 2px solid grey;
+    border-radius: 5px;
+    margin:20px;
+
+}
+
+ #card img{
+    width: -webkit-fill-available;
+} 
+
+#header,#header a{color:#fff}
+
+#header h1{
+    margin: 0px;
+    padding: 10px;
+}
+
+#header{
+    background-color: black;
+    display:flex;
+    justify-content: space-between;
+    align-items:center;
+    padding: 5px 30px;
+    font-family: monospace;
+}
+
+#search-bar{
+    width: 300px;
+    margin: 10px;
+    padding: 10px;
+    border-radius: 5px;
+}
+
+form{
```

Comment:
Keep *, body, html tag selectors on top of the CSS code to follow CSS specificity.

Reply:
fixed